.github/workflows/ci.yml

```diff
@@ -222,3 +222,51 @@ jobs:
         if: ${{ matrix.server == 'cloud' && env.TEMPORAL_CLIENT_CERT != '' }}
       - run: rm /tmp/temporal-certs/client.key
         if: ${{ matrix.server == 'cloud' && env.TEMPORAL_CLIENT_CERT != '' }}
+
+  # Run the sdk features repo tests with this repo's current SDK code
```

Comment:
@bergundy Do you have an example of how you trigger it?

AFAICT cross-repo workflow calling isn't super well supported in github actions. https://docs.github.com/en/actions/using-workflows/reusing-workflows looks like it might work for it, but has quite a few limitations, or like literally posting `CURL` requests to the other repo which would have issues with visibility in the triggering repo.

Given how little code this actually is I'm not sure it's really worth making compromises just to re-use a bit. All the real work should be performed in the harness itself anyway

Reply:
See here: https://github.com/marketplace/actions/repository-dispatch and here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch

Reply:
Lol
>A GitHub action to create a repository dispatch event.

Is about the least useful description I have ever read.

In any case, if I understand this right, I think it's much worse than this solution except for some definition re-use. This will go trigger some job on the features repo in a fire-and-forget manner, but we want to have those tests run as a first class citizen right in here, and fail the PR if they don't pass.

Reply:
You can always have the sdk-features workflow post a status back

Reply:
Sure, but, that seems much less usable, for little benefit? Just saving a few lines of setup.

Reply:
I want to avoid the duplication since the sdk-features CI is going to also need to run the TS harness.
AFAIK, we have 2 options for avoiding duplication:
1. Triggering the sdk-features build with parmeters from TS and blocking on status
2. Sharing a workflow definition between the 2 repos (e.g. what I started doing [here](https://github.com/temporalio/sdk-typescript/pull/395/files))

Reply:
Option 2 is working well here looks like